.github/workflows/deploy-customer.yml

```
@@ -46,83 +40,21 @@ jobs:
+          source: "eatngo-customer-api/build/libs/*.jar"
+          target: "/home/ec2-user/eatngo-customer-api"
+
+      - name: Remote deploy on EC2
+        uses: appleboy/ssh-action@v1.0.3
+        with:
+          host: ${{ secrets.EC2_HOST }}
+          username: ec2-user
+          key: ${{ secrets.EC2_SSH_KEY }}
+          script: |
+            pkill -f 'eatngo-customer-api' || true
+            nohup java -jar /home/ec2-user/eatngo-customer-api/*.jar --spring.profiles.active=dev > app.log 2>&1 &
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)